contracts/examples/multisig/mandos/upgrade.scen.json

```
@@ -0,0 +1,166 @@
+{
+    "gasSchedule": "v3",
+    "steps": [
+        {
+            "step": "externalSteps",
+            "path": "deployOtherMultisig.scen.json"
+        },
+        {
+            "step": "scCall",
+            "txId": "proposeSCUpgrade",
+            "comment": "missing argument",
+            "tx": {
+                "from": "address:paul",
+                "to": "sc:multisig",
+                "function": "proposeSCUpgrade",
+                "arguments": [
+                    "sc:multisig-child",
+                    "0",
+                    "file:../test-contracts/adder.wasm",
+                    "false",
+                    "false",
+                    "false",
+                    "1234"
+                ],
```

Similar situation here. The contract which is upgraded here is then used in `upgrade_from_source.scen.json`.
It should be upgradable:
```suggestion
                "function": "proposeSCUpgrade",
                "arguments": [
                    "sc:multisig-child",
                    "0",
                    "file:../test-contracts/adder.wasm",
                    "true",
                    "false",
                    "false",
                    "1234"
                ],
```